limiter/alimiter.go

```diff
@@ -35,31 +35,31 @@ func getWeighted(n, max int) int {
 
 // ALimiter provide limiter amount of requests/concurrently executing requests (adaptive with load avg)
 type ALimiter struct {
-	l  limiter
-	cL limiter
-	c  int
-	n  int
+	l   limiter
```

Comment:
Maybe we should improve naming of these fields? It's hard to understand what do they represent